sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultKeyManagerFactory.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.jca;
+
+import javax.net.ssl.KeyManager;
+import javax.net.ssl.KeyManagerFactorySpi;
+import javax.net.ssl.ManagerFactoryParameters;
+import java.security.KeyStore;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+/**
+ * The KeyVault variant of the KeyManagerFactory.
+ */
+public class KeyVaultKeyManagerFactory extends KeyManagerFactorySpi {
+
+    /**
+     * Stores the logger.
+     */
+    private static final Logger LOGGER = Logger.getLogger(KeyVaultKeyManagerFactory.class.getName());
+
+    /**
+     * Stores the key managers.
+     */
+    private final List<KeyManager> keyManagers = new ArrayList<>();
+
+    @Override
+    protected void engineInit(KeyStore keystore, char[] password) {
```

Comment:
why protected and not public ?

Reply:
Not my choice this is part of the KeyStoreSpi delivered as part of JCA